backend/src/test/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceControllerTest.java

```diff
@@ -42,27 +43,27 @@
 
   @Autowired private PatientExperienceController _controller;
```

Comment:
Do we want to keep the underscore pattern for autowired fields?

Reply:
oh great catch! yeah that needs to be also changed